### PR TITLE
[ADD] openacademy: Unitest about course and session constrains were a…

### PR DESCRIPTION
**Pull request by CDanielSanSeg**
…dded. Files for unitest are:

    - open_academy/tests/__init__.py
    - open_academy/tests/test_global_openacademy_course.py
    - open_academy/tests/test_global_openacademy_session.py
    - open_academy/views/open_academy_session_views.xml

### CODE REVIEW

**Comment by CDanielSanSeg** [comment]
Hi @edy1192, can you check this PR?

[ADD] openacademy: Unitest about course and session constrains were added. Files for unitest are:

    - open_academy/tests/__init__.py
    - open_academy/tests/test_global_openacademy_course.py
    - open_academy/tests/test_global_openacademy_session.py
    - open_academy/views/open_academy_session_views.xml

File: open_academy/models/open_academy_course.py [hunk 1]
```diff
@@ -29,3 +29,6 @@ def copy(self, default=None):
         default = dict(default or {})
         default.update({'title': f'Copy of {self.title}'})
         return super().copy(default=default)
+
+    def algo(self):
+        pass
```
Comment (edy1192) [inline open_academy/models/open_academy_course.py]: Why this?

File: open_academy/tests/test_global_openacademy_session.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+from odoo.exceptions import ValidationError
+from odoo.tests.common import TransactionCase
+
+
+class GlobalTestOpenAcademyCourse(TransactionCase):
```
Comment (edy1192) [inline open_academy/tests/test_global_openacademy_session.py]:
```suggestion
class GlobalTestOpenAcademySession(TransactionCase):
```

**Review by edy1192 — CHANGES_REQUESTED** [review]

**Comment by CDanielSanSeg** [comment]
@edy1192 can you check the changes in this PR?

**Review by edy1192 — APPROVED** [review]

**Pull request merged**